tests/test_httperror.py

```diff
@@ -583,15 +583,15 @@ def test_405_without_body(self, client):
 
         response = client.simulate_request(path='/405')
         assert response.status == falcon.HTTP_405
-        assert not response.content
+        assert response.text == falcon.HTTPMethodNotAllowed(['PUT']).to_json()
```

Comment:
Do we have assertions against the actual response content?
The above approach is cool and DRY, but if there was a bug in `to_json()`, this would pass. In other words, this test is just verifying that the `to_json()` method is called, and the call result is transferred to the response body.

Reply:
Yes, this test does only that. It assumes that `.to_json()` is correct.
`.to_json()` should be already tested somewhere else. 

In case it's missing I can add that test, but I would not retest it here, or this test (all the other similar ones) become overly broad

Reply:
As long as `to_json()` is tested elsewhere for `HTTPMethodNotAllowed`, it's fine.

Reply:
It seems not. I'll test in one tests